pkg/graph/resolvers/trb_request_feedback.go

```diff
@@ -0,0 +1,87 @@
+package resolvers
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/models"
+	"github.com/cmsgov/easi-app/pkg/storage"
+)
+
+// CreateTRBRequestFeedback creates a TRBRequestFeedback in the database
+func CreateTRBRequestFeedback(ctx context.Context, store *storage.Store, feedback *models.TRBRequestFeedback) (*models.TRBRequestFeedback, error) {
+	feedback.CreatedBy = appcontext.Principal(ctx).ID()
```

Comment:
I notice that this function calls appcontext.Principal, and appcontext.Logger quite a few times. It might make sense to just store them as variables instead of calling the functions multiple times.